### PR TITLE
patch CMakeLists

### PR DESCRIPTION
**Pull request by mefuller**
Modify CMakeLists.txt to place files in correct lib dir (e.g. on lib64 systems)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -78,7 +78,7 @@ install(TARGETS maeparser
 
 INSTALL(EXPORT maeparser-targets
     FILE ${PROJECT_NAME}-config.cmake
-    DESTINATION lib/cmake)
+    DESTINATION lib${LIB_SUFFIX}/cmake)
```
Comment (d-b-w) [inline CMakeLists.txt:81]: I don't know cmake very well... But would it be more idiomatic to use `CMAKE_INSTALL_LIBDIR`, instead?

**Review by d-b-w — CHANGES_REQUESTED** [review]
What's the symptom of the current problematic configuration? Is this the most idiomatic way to address the symptom?

File: CMakeLists.txt (hunk 1 above)
Comment (mefuller) [reply CMakeLists.txt:81]: I also do not really know CMAKE well.
The symptom is that unpatched, the file installs to `/usr/lib` on Fedora 64-bit systems.

I am the packer for maeparser on Fedora, and for compliance with our guidelines, the location must be under `/usr/lib64` on 64-bit systems.

The patch, as submitted, accomplishes this goal.

Comment (ricrogz) [reply CMakeLists.txt:81]: I think Dan's suggestion makes sense: right above this block, we set `CMAKE_INSTALL_LIBDIR` as the install directory for libraries in the `maeparser` target. I think it makes sense to put the `cmake` directory there too:
```suggestion
    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
```

This should also work, and is more coherent with the other definitions.

**Review by ricrogz — APPROVED** [review]
Thanks!

**Comment by ricrogz** [comment]
Hm, I think we need to update some of the CI build instructions

**Comment by mefuller** [comment]
thanks @ricrogz and @d-b-w 

for reference, the Fedora package is visible at https://src.fedoraproject.org/rpms/maeparser

**Comment by d-b-w** [comment]
Thanks for this, @mefuller

By the way - who writes the package documentation? Is there some metadata that we should add here that would make it easier for package managers like yourself to automatically generate the documentation? It looks like it's independently written, but based on the README.

**Comment by mefuller** [comment]
Independently generated - I copied it in

**Pull request merged**